src/components/Hello.vue

```diff
@@ -2,10 +2,17 @@
   <md-layout md-column>
     <md-toolbar>
       <h2 style="flex: 1" class="md-title">Greeting console</h2>
+
       <md-switch @change="switchOnline" v-model="online" id="my-test0"
                  name="my-test0"></md-switch>
       <span v-if="online">Online</span>
       <span v-else>Offline</span>
+
+      <span style="margin-left: 20px; width: 100px" v-if="online">
```

Comment:
Not pretty, but does the job.

![image](https://user-images.githubusercontent.com/1097193/154800725-81dddeef-8cf5-45c4-8b86-0711cf8c7403.png)

Could be improved though